game/templates/game/board.html

```
@@ -638,813 +688,93 @@ <h1>Checkora <span class="badge" id="modeBadge">PVP</span></h1>
     <div class="promo-overlay" id="gameOverOverlay">
         <div class="promo-dialog" style="min-width: 320px; padding: 35px; text-align: center;">
-            <div class="promo-title" id="gameOverTitle" style="font-size: 1.5rem; color: #f0c040; margin-bottom: 10px;">Game Over</div>
+            <div class="promo-title" id="gameOverTitle" style="font-size: 1.5rem; color: #f0c040; margin-bottom: 10px;">
+                Game Over</div>
             <p id="gameOverMessage" style="color: #ccc; margin-bottom: 25px; font-size: 1rem;"></p>
+            <div style="margin-bottom: 20px;">
+                <label style="color: #ccc; display: block; margin-bottom: 8px; font-size: 0.9rem;">Game Mode</label>
+                <div style="display: flex; gap: 10px; margin-bottom: 15px;">
+                    <label
+                        style="flex: 1; cursor: pointer; background: #1a1a2e; padding: 10px; border-radius: 5px; border: 1px solid #444; text-align: center; color: #fff; font-size: 0.9rem;">
+                        <input type="radio" name="go_mode" value="pvp" checked
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)